pages/home/release-notes.mdx

```diff
@@ -41,6 +41,44 @@ The Allora Network v0.5.0 is now live! This version introduces several major upd
 #### Idempotent Payload Submission
 - Simplified submission conditions for inference payloads. Duplicate submissions are now handled in an idempotent manner, preventing them from affecting the system’s behavior or causing erroneous calculations.
 
+## v0.4.0
```

Comment:
might want to sneak v0.5.0 into this as well, but also do we really need to have the changelog in two places? We already have a changelog in the repo, maybe we can just link to it here